source/parser.hera

```diff
@@ -919,20 +932,21 @@ NestedBindingProperties
     }
 
 NestedBindingProperty
-  Nested:indent BindingProperty:prop ->
-    return {
+  Nested:ws ( BindingProperty ObjectPropertyDelimiter )+:inlineProps ->
```

Comment:
Seems like `( BindingProperty ObjectPropertyDelimiter )+` should be replaced by equivalent `BindingPropertyList`. Oh, you're doing something different in the code block. But maybe just the whitespace insertion could be done in this code block, and avoid duplicating code?